File: PRAKTIKUM 1/PRAKTIKUM 1/C/PRAK110-2210817110002-MUHAMMAD PASHA NABEEL.c
#include<stdio.h>

int main()
{
    int a,t,m,K,L;
    a = 5;
    t = 12;
    m = sqrt(pow(a,2)+pow(t,2));
    K = a+t+m;
    L = a*t/2;

    printf("Diketahui : \n");
    printf("Alas = %d cm\nTinggi = %d cm\n\n", a, t);

    printf("Jawab :\n");
    printf("Sisi A = %d cm\n", t);
    printf("Sisi B = %d cm\n", m);
    printf("Sisi C = %d cm\n\n", a);
    printf("Keliling = %d cm\n", K);
    printf("Luas = %d cm\n", L);
    return 0;
}

File: PRAKTIKUM 1/PYTHON/PRAK110-2210817110002-MUHAMMAD PASHA NABEEL.py
import math
a = 5
t = 12
m = math.sqrt(pow(a,2)+pow(t,2))
K = a+t+m
L = a*t/2

print("Diketahui : ")
print("Alas = %d cm\nTinggi = %d cm\n"% (a, t))

print("Jawab :")
print("Sisi A = %d cm"% t)
print("Sisi B = %d cm"% m)
print("Sisi C = %d cm"% a)
print("Keliling = %d cm"% K)
print("Luas = %d cm"% L)